copy-to-apps.sh

```diff
@@ -2,24 +2,4 @@
 # Copyright 2017-2020 @polkadot/api authors & contributors
 # SPDX-License-Identifier: Apache-2.0
 
-function copy_folder () {
-  SRC="packages/$1/build"
-  DST="../apps/node_modules/@polkadot/$1"
-
-  echo "** Copying $SRC to $DST"
-
-  rm -rf $DST
-  cp -r $SRC $DST
-}
-
-yarn polkadot-dev-build-ts
-
-copy_folder "api"
-copy_folder "api-contract"
-copy_folder "api-derive"
-copy_folder "metadata"
-copy_folder "rpc-core"
-copy_folder "rpc-provider"
-copy_folder "types"
-copy_folder "types-known"
-copy_folder "typegen"
+./copy-to.sh "apps"
```

Comment:
This should be reverted. There is a more generic version that will be added to dev. (Since it is duplicated everywhere)